backends/tc/tc.def

```diff
@@ -278,19 +292,6 @@ class TCTable {
                 }
             }
         }
-
-        if (preaction != nullptr) {
```

Comment:
Is this related to default action or some stale code being removed now?

Reply:
Stale code removed now